demo/Meter.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Meter Styling</title>
+		<link rel="stylesheet" href="../dist/css/chassis.css">
+	</head>
+	<body>
+		<meter max="100" value="50" class="meter"></meter>
+		<br>
```

Comment:
You're using way too many `<br>` here.  Maybe adjust the natural spacing of a meter?